website/docs/d/list_profiles.html.markdown

```diff
@@ -0,0 +1,84 @@
+---
```

Comment:
These file names should be changed as well to scc_posture_profiles.. basically from the resource name which is

ibm_scc_posture_profiles drop the ibm  so the above file will become scc_posture_profiles.html.markdown. Check other documention files as well